src/astrobleme/gui/Painter.java

```diff
@@ -58,7 +58,7 @@ public static void paint(Grid grid, GraphicsContext gc) {
 
         // The score
         gc.setFill(Color.BEIGE);
-        gc.fillText("Score : " + 100 * snake.getPoints().size(), 10, 490);
+        gc.fillText("Score : " + 100 * snake.getPoints().size(), 10, 10);
```

Comment:
Again, this is a relatively large change in position. It might be better to use an environment variable with a default fallback value here.